website/playground/src/romeWorker.ts

```diff
@@ -25,31 +75,96 @@ self.addEventListener("message", async (e) => {
 				isJsx,
 				sourceType,
 				cursorPosition,
-			} = e.data.playgroundState;
-			const romeOutput = run(
-				code,
-				new PlaygroundFormatOptions(
-					lineWidth,
-					indentStyle === IndentStyle.Space ? indentWidth : undefined,
-					quoteStyle,
-				),
-				isTypeScript,
-				isJsx,
-				sourceType,
-				cursorPosition,
-			);
-			self.postMessage({
-				type: "formatted",
-				romeOutput: {
-					ast: romeOutput.ast,
-					cst: romeOutput.cst,
-					errors: romeOutput.errors,
-					formatted_code: romeOutput.formatted_code,
-					formatter_ir: romeOutput.formatter_ir,
-					control_flow_graph: romeOutput.control_flow_graph,
+			} = playgroundState;
+
+			workspace.update_settings({
+				settings: {
+					format: {
+						enabled: true,
+						format_with_errors: true,
+						line_width: lineWidth,
+					},
+					linter: {
+						enabled: true,
+					},
+					languages: {
+						javascript: {
+							format: {
+								quote_style: quoteStyle === QuoteStyle.Double ? "Double" : "Single",
```

Comment:
Should we use the enum instead of the strings? Less prone to typos 

```suggestion
								quote_style: quoteStyle === QuoteStyle.Double ? QuoteStyle.Double : QuoteStyle.Single,
```

Reply:
Unfortunately because TypeScript doesn't resolve the value of enums during compilation the `QuoteStyle.Double` expression and `import { QuoteStyle } from 'rome_playground'` statement would remain in the emitted code. This means the bundler will expect the `rome_playground` module have an export named `QuoteStyle`, but the JS code generated by `wasm-bindgen` doesn't include this